src/components/elements/DomainImage.tsx

```diff
@@ -3,13 +3,14 @@ import { Flex } from "rimble-ui";
 
 export interface DomainImageProps {
   domain: string;
+  size?: string;
 }
 
 export const DomainImage: React.FunctionComponent<DomainImageProps> = (props) => {
   return (
-    <Flex alignItems="center" width="16px" {...props}>
+    <Flex alignItems="center" width={props.size || "16px"} {...props}>
       <img
-        src={"https://www.google.com/s2/favicons?domain=" + props.domain}
+        src={"https://www.google.com/s2/favicons?sz=64&domain=" + props.domain}
```

Comment:
We can try this out but I think most favicons will look pixelated at 64. I've noticed a lot are only 16px or 32px